discovery/install/install.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package install has the side-effect of registering all builtin
+// service discovery config types.
+package install
+
+import (
+	_ "github.com/prometheus/prometheus/discovery/azure"        // register azure
```

Comment:
The comments aren't needed.

Reply:
They're there to make golint happy, otherwise: `a blank import should be only in a main or test package, or have a comment justifying it`